### ISSUE TITLE
Unclosed bracket error in url

### ISSUE THREAD

**Issue by ansee66** [postcss/postcss#1913]
I ran into a problem when building the project and in search of an answer I found this [issue](https://github.com/postcss/postcss-scss/issues/142). 
After testing, I found out that I have the same problem - an error occurs when parsing if there is any value before "url" in the background property. 
For example ("no-repeat bottom center" in this case): 
```
background: url("https://example.com/test.png") no-repeat bottom center, url("https://example.com/test.png") no-repeat top center;
```
As a result, the question arose: is it possible to fix the error in this repository, as suggested by the author of that issue? [His PR](https://github.com/postcss/postcss-scss/pull/143/commits/0eb14c90462c2104d8677cf5307690fb3e1e52f4#diff-bb5b4ae074d4e589fb803fcca88860248753c914134334ae54fb09205fa8a33e) to the postcss-scss repository

**Comment by ai** [comment]
I didn’t find any issue in PostCSS core parser (for vanilla CSS).

```js
let postcss = require('postcss')

let css = `
a {
  background: url("https://example.com/test.png") no-repeat bottom center, url("https://example.com/test.png") no-repeat top center;
}`

console.log(postcss.parse(css).first.first.value)
//=> url("https://example.com/test.png") no-repeat bottom center, url("https://example.com/test.png") no-repeat top center
```

That issue you mentioned was only in `postcss-scss`, nothing need to be fixed here.

Maybe your issue is related to some tool around PostCSS ecosystem?

1. What is the full error message with error stack trace?
2. Do you have vanilla CSS or Sass?

**Comment by ansee66** [comment]
Perhaps you are right and this error is in some other tool - we use next.js and scss modules.
Error stack trace:
![2024-01-13_22-34-31](https://github.com/postcss/postcss/assets/65064128/68a52f61-47be-47e5-96d5-77894a7966b0)

**Comment by ai** [comment]
What version of `postcss-scss` do you use? Check in npm lock file.

Maybe you are using the version before the fix?

**Comment by ansee66** [comment]
In package.json of next ("version": "14.0.4") "postcss-scss": "4.0.3" is specified. Is this version before the fix?

**Comment by ai** [comment]
Yes, the problem is in old version.

That issue, which you found, was fixed in 4.0.6. The latest version is 4.0.9.
